src/models/converters/transactions/tests/check_sender_or_receiver.rs

```diff
@@ -0,0 +1,112 @@
+use crate::models::commons::{DataDecoded, Parameter};
+use crate::models::converters::transactions::check_sender_or_receiver;
+use crate::models::commons::ParamValue::SingleValue;
+
+#[test]
+fn check_sender_or_receiver_safe_sender() {
+    let data_decoded = Some(DataDecoded {
+        method: "transfer".to_string(),
```

Comment:
Not sure what we test here, as for a `transfer` method the params are not important

Reply:
So this is just one of the 3 happy paths. I don't understand very well what this method is supposed to do, but it seems that it checks that data decoded contains a certain address value in a parameter that's either called `from` or `to` and also that the `method` field is `transfer`, any of these 3 is true then this method returns true, which from a business logic perspective, I don't understand exactly what that means.